js/build_map.js

```diff
@@ -180,7 +180,7 @@ function fillMap() {
   // load county data, add and update county polygons.
   // it's OK if it's not done right away; it should be loaded by the time anyone tries to hover!
   // and it doesn't need to be done at all for mobile
-  if(waterUseViz.mode !== 'mobile') {
+  if(waterUseViz.viewport !== 'narrow') {
```

Comment:
this one should be `if(waterUseViz.interactionMode === "tap")`

Reply:
I'm confused on this one cause you are making sure it's not mobile, so !== 'tap'?